### PR TITLE
Feat/map update 

### PR DESCRIPTION
**Pull request by Leftere**
Issue #382 

I have restyled the Google map to fit our design, I also got rid of lots of unnecessary default labels and icons to improve visual aspect.

 - Please let me know if you like it or if you have any other suggestions.

- I also added a temporary new icon.

My plan is to give each institution a uniq icon so that the users knows what they click on.

### CODE REVIEW

**Comment by johnpyp** [comment]
Screenshots of the change: https://imgur.com/a/C1w75Tx

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+      "language": "html",
+      "autoFix": true
+    }
+  ],
+  "eslint.enable": true,
+  "eslint.options": {
+    "extensions": [".html", ".js", ".vue", ".jsx"]
+  },
+  "eslint.run": "onSave",
+  "eslint.autoFixOnSave": true
```
Comment (davidholyko) [inline .vscode/settings.json]: dont commit this file

File: key_file [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Yu8klhCwk1n3pog9W+Yll3WD5NrOZu4LLLBWPt5j7qfy5/qySNK2uAULIFNyOxih
+NauZwV4FzAN32C1a1azgl5+EVmcUj9O+fHkhUNJIwqql35GED8/3JyYSQycb0Ca/
+LI+B9hH44ySvISLWanTxbEbJXzYAWeVt9DzNFE3s1scdUUONeiDzUVg9qnNiu2d+
+tPnCl/BHIr2uRKuZO+Itm1zwlcQXvwrayzgUCDSlrKntbl5AsoYH7Qj2UgXAkrHY
+tRPicxyfIi9b4zbwbkO4kBXF+tYKhhVW14/IT4oHhGuxFjR+YBIFnBkn4wIT5Yea
+lJpoVSHRXvi1LO8OLgt18RxeEGmt8i4/xE9D/7W/vJ/aXTvDYtBs4qoAZcg/E6hs
+om4r62Iuet7ByFWVF9OrZtcHOukk4WcZyQOxNWehTjZ87vdGSymqFeP5CIGncCKy
+03MGfXxKKvZa3kTcY6jW6DUJIm8QR7Ik9N8FN4bJcAmOWm6CgNoSyYVlW4TVjy/w
+mHC29LEtex2g8bRaQGl/dTnrHlOM1woV34KrpUU+0fzsJW7B+0IrRqtybgh4hktR
+-----END RSA PRIVATE KEY-----
```
Comment (davidholyko) [inline key_file]: I dont think this key is used anywhere in this code.

File: key_file.pub [hunk 3]
```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCi3kUTVjIVhkaa0GSr3vieErzdaVlZs1tski+XwGQZFFq2jCHfxHl361bLx6pQRwak/QQRZ6soPveHEvOxm/Ss7nMZS34twni3M709uAjanaFVx7hjiRjf+YLiAyyCWlsTt/1QPjMdSl3NUWiK5TUaR3qmOKI6utmqMwB2wT/5Py5wq/zSMcQ7BspytztkTug2o3eKvD7OhWvDfhtjhWgCybk1rETbVdxzbvsqqFsqqWX6YJII97IHlB5VoJBjm7TYopBS+amh4enFTwBkI8wzRuKdONoZ2sR138J1zMN9s4vEe9So1OInshbYM3JISCIeGtIZi5eaJP14N5ewj5K8Pi3i+4rP619f5ZUp4L9qbIKz6VGwmg4xEDc3ARYYFxpsTJz4+wLmmPadW7RqI528cwD/cfLmeO56pli7hviBT0itc9w2GFAhYFHm91EQBsTvkZTSWThXdx9Tt9vl3Cj1B9hpaHwaf8X99aln7NxMzmif6qr/46XwYGa4gVolBDX3fEbdbWXrUHpXMEwP24PpZBfq5QXVx8n9iEvaGEzbVUVwhTVcJrV3Ogt5C+X9AaUfu0ie+fiRW1grcfuk+R1VW2ZvZSbsIJeBhRP8n2bqBfrGF26IMJpkGGJEqyWZbbN+dApfeg4MUQaABbN8nYc9AkyqIFhs37Cj+hLzSqg3Lw== eugen.lefter@gmail.com
```
Comment (davidholyko) [inline key_file.pub]: i think this key is also not used

File: src/components/MapPage/Map.js [hunk 4]
```diff
@@ -1,34 +1,229 @@
...
-import { MarkerClusterer } from 'react-google-maps/lib/components/addons/MarkerClusterer';
-import OrganizationMarker from './OrganizationMarker';
-import { compose, lifecycle } from 'recompose';
+import React from "react";
+import { withScriptjs, withGoogleMap, GoogleMap } from "react-google-maps";
+import { MarkerClusterer } from "react-google-maps/lib/components/addons/MarkerClusterer";
+import OrganizationMarker from "./OrganizationMarker";
+import { compose, lifecycle } from "recompose";
+
+const styles = [
```
Comment (davidholyko) [inline src/components/MapPage/Map.js]: this styles is a little big. You might want to break it off into its own file and import in

GoogleMapConfig.js
```
export const googleMapConfig = [...]

```

Map.js
```
import {googleMapConfig} from './GoogleMapConfig'

// ...

<GoogleMap
...
defaultOptions={googleMapConfig}

```

File: src/components/MapPage/Map.js [hunk 5]
```diff
@@ -1,34 +1,229 @@
...
+    {...props}
+    ref={props.zoomToMarkers}
+    defaultZoom={5}
+    defaultOptions={{
+      styles: styles
+    }}
+  >
     <MarkerClusterer
       averageCenter={true}
+      // icon={iconMarker}
```
Comment (davidholyko) [inline src/components/MapPage/Map.js]: remove unused commented code

File: src/index.js [hunk 6]
```diff
@@ -1,26 +1,27 @@
...
+import "mapbox-gl/dist/mapbox-gl.css";
+import "./css/index.scss";
 
 library.add(fab, fas);
 
 export const getRoutes = store => {
   return (
     <div>
       <Switch>
+        {/* <Route exact path="/" component={MyHomePageComponent} /> */}
```
Comment (davidholyko) [inline src/index.js]: remove unused commented code

File: src/components/MapPage/Map.js (hunk 4 above)
Comment (Leftere) [reply src/components/MapPage/Map.js]: agree

File: key_file (hunk 2 above)
Comment (Leftere) [reply key_file]: I'm not sure what this key is either, maybe my SSH key?

**Comment by johnpyp** [comment]
If that is your SSH key, I'd recommend changing it right away. There are bots that scrape github for things like passwords, other credentials, and ssh keys, so they could use it to access your github or anything your ssh key is authenticated with.

**Comment by RajChaudhry** [comment]
I don't have any review comments that haven't been left by others. When you say "give each institution a unique icon" do you mean each resource? Because there are currently around resources, 100, and as we grow there are gonna be a lot more. That'll be tough to sustain and onboard.

**Comment by Leftere** [comment]
Yes, each resource.

**Comment by Leftere** [comment]
Ok guys, I think I fixed all your comments and Im ready to merge this. Let me know If I need to do anything. 

For some reason I'm under impression that I need to close this pull request and create a new one.
I am sure I am wrong though

**Comment by RajChaudhry** [comment]
> Ok guys, I think I fixed all your comments and Im ready to merge this. Let me know If I need to do anything.
> 
> For some reason I'm under impression that I need to close this pull request and create a new one.
> I am sure I am wrong though

@Leftere It doesnt seem like you pushed your changes to this branch. Are you sure you added + committed all the right files? And that you pushed to the feat/map-update branch?

**Pull request closed without merging**